pom.xml

```diff
@@ -35,7 +35,7 @@
         <dependency>
             <groupId>org.jenkins-ci.plugins</groupId>
             <artifactId>scm-api</artifactId>
-            <version>2.0</version>
+            <version>2.0.1-SNAPSHOT</version>
```

Comment:
No, these were changes of sufficiently low risk that I just merged them... you can review if you want: https://github.com/jenkinsci/scm-api-plugin/compare/scm-api-2.0...master